redis/src/main/scala/zio/redis/api/Strings.scala

```diff
@@ -10,68 +10,116 @@ import zio.{ Chunk, ZIO }
-
+    key: String,
+    firstCommand: BitFieldCommand,
+    restCommands: BitFieldCommand*
+  ): ZIO[RedisExecutor, RedisError, Chunk[Option[Long]]] = BitField.run((key, (firstCommand, restCommands.toList)))
+
+  // Perform bitwise operations between strings
+  final def bitOp(
+    operation: BitOperation,
+    destKey: String,
+    firstSrcKey: String,
+    restSrcKeys: String*
+  ): ZIO[RedisExecutor, RedisError, Long] =
+    BitOp.run((operation, destKey, (firstSrcKey, restSrcKeys.toList)))
+
+  // Find first bit set or clear in a string
+  final def bitPos(key: String, bit: Boolean, range: Option[BitPosRange] = None): ZIO[RedisExecutor, RedisError, Long] =
+    BitPos.run((key, bit, range))
+
+  // Decrement the integer value of a key by one
+  final def decr(key: String): ZIO[RedisExecutor, RedisError, Long] = Decr.run(key)
+
+  // Decrement the integer value of a key by the given number
+  final def decrBy(key: String, decrement: Long): ZIO[RedisExecutor, RedisError, Long] = DecrBy.run((key, decrement))
+
+  // Get the value of a key
+  final def get(key: String): ZIO[RedisExecutor, RedisError, Option[String]] = Get.run(key)
+
+  // Returns the bit value at offset in the string value stored at key
+  final def getBit(key: String, offset: Long): ZIO[RedisExecutor, RedisError, Long] = GetBit.run((key, offset))
+
+  // Get a substring of the string stored at key
+  final def getRange(key: String, range: Range): ZIO[RedisExecutor, RedisError, String] = GetRange.run((key, range))
+
+  // Set the string value of a key and return its old value
+  final def getSet(key: String, value: String): ZIO[RedisExecutor, RedisError, Option[String]] =
+    GetSet.run((key, value))
+
+  // Increment the integer value of a key by one
+  final def incr(key: String): ZIO[RedisExecutor, RedisError, Long] = Incr.run(key)
+
+  // Increment the integer value of a key by the given amount
+  final def incrBy(key: String, increment: Long): ZIO[RedisExecutor, RedisError, Long] = IncrBy.run((key, increment))
+
+  // Increment the float value of a key by the given amount
+  final def incrByFloat(key: String, increment: Double): ZIO[RedisExecutor, RedisError, String] =
+    IncrByFloat.run((key, increment))
+
+  // Get all the values of the given keys
+  final def mGet(firstKey: String, restKeys: String*): ZIO[RedisExecutor, RedisError, Chunk[Option[String]]] =
+    MGet.run((firstKey, restKeys.toList))
+
+  // Set multiple keys to multiple values
+  final def mSet(
+    firstKeyValue: (String, String),
+    restKeyValues: (String, String)*
+  ): ZIO[RedisExecutor, RedisError, Unit] =
+    MSet.run((firstKeyValue, restKeyValues.toList))
+
+  // Set multiple keys to multiple values only if none of the keys exist
+  final def mSetNx(
+    firstKeyValue: (String, String),
+    restKeyValues: (String, String)*
+  ): ZIO[RedisExecutor, RedisError, Boolean] =
+    MSetNx.run((firstKeyValue, restKeyValues.toList))
+
+  // Set the value and expiration in milliseconds of a key
```

Comment:
In order to have a valid scaladoc comment, you should use `/** <comment here> */` format as you've already used below. Also, try using `@param` to describe method parameters and `@return` to describe a return value. I suggest that you don't use `@return` when the returned ZIO effect has a value of type `Unit`.

Here is one example:
```suggestion
 /** Sets a value and expiration in milliseconds of the specified key.
  *
  *  @param key Redis key identifier
  *  @param milliseconds expire time that is expressed in milliseconds
  *  @param value key value to be set
  */
```

For more details you can read the [documentation](https://docs.scala-lang.org/style/scaladoc.html).

Reply:
Sure I will update it shortly.